opflexagent/test/test_endpoint_file_manager.py

```diff
@@ -44,6 +45,9 @@ def setUp(self):
         self._mock_agent(self.manager)
         self.addCleanup(self._purge_endpoint_dir)
 
+        ip_lib.IPDevice = mock.Mock()
+        ip_lib.IpLinkCommand = mock.Mock()
```

Comment:
Is the mock of IpLinkCommand  needed?

Reply:
You are right, it's not needed. I will remove it.